advancedbrowser/advancedbrowser/consts.py

```diff
@@ -0,0 +1,21 @@
+"""Some values that are in anki/consts (in master) but not yet in 2.1.21"""
```

Comment:
I'm importing the constants now that we're targeting the next release for 2.1.24, so there's no need for this.

Reply:
I didn't expect 22 and 23 to come out so quickly when I made this PR. I admit this became useles. But honestly, it took me more than a year before Damien accepts my PR adding those constants in anki